nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/marshallers/DocumentMarshaller.java

```diff
@@ -91,10 +92,12 @@ protected static Document readDocument(JsonParser jp) throws IOException {
                 readProperties(jp, properties);
             } else if ("changeToken".equals(key)) {
                 changeToken = jp.getText();
+            } else if ("isProxy".equals(key)) {
+                isProxy = jp.getBooleanValue();
```

Comment:
Seems that `jp.getBooleanValue()` returns a `boolean` so you can simplify to use `boolean` everywhere.